test/chat_api/reporting_test.exs

```diff
@@ -42,6 +42,38 @@ defmodule ChatApi.ReportingTest do
              ] = Reporting.messages_by_date(account.id)
     end
 
+    test "count_messages_per_user/1 should return correct number of messages sent per user on team",
+         %{
+           account: account,
+           customer: customer
+         } do
+      account_2 = account_fixture()
+      account_3 = account_fixture()
```

Comment:
these extra accounts shouldn't be necessary, we can just add users to the existing `account` from the setup :)